moveit_core/collision_detection/src/collision_common.cpp

```diff
@@ -46,7 +46,7 @@ namespace
 {
 rclcpp::Logger getLogger()
 {
-  return moveit::getLogger("collision_detection_common");
+  return moveit::getLogger("moveit.core.collision_detection_common");
```

Comment:
Following the pattern you mentioned in the description `moveit.<package>.<context>.<optional detail>`, should this be renamed to 

```suggestion
  return moveit::getLogger("moveit.core.collision_detection.collision_common");
```